mgcpy/independence_tests/utils/compute_distance_matrix.py

```diff
@@ -0,0 +1,8 @@
+def compute_dist(matrix_X, matrix_Y, compute_distance):
```

Comment:
Give full names to variables, that is how we have been following throuoght the package, `compute_distance`, would be better.